README.md

```diff
@@ -1,29 +1,40 @@
-[![Ubuntu CMake Build](https://github.com/learn-orolia/skydel-plug-ins/actions/workflows/ubuntu_cmake_build.yml/badge.svg)](https://github.com/learn-orolia/skydel-plug-ins/actions/workflows/ubuntu_cmake_build.yml)
```

Comment:
Idk if we should be deleting the repo's original readme. I think we should keep their readme here, and add another readme for our ublox plugin (either with a different name in this same location, or just a README.md in the directory of our plugin. Another option would be to add our text to the bottom of their README.

Reply:
Felix told us a bunch of times to change the README to reflect our plugin, but you might still be right. It might be better to move theirs to the bottom but still keep it.

Reply:
@Rouiller do you want our main README.md to be reflective of our plugin? Or should we have this in the ublox-plugin directory?

Reply:
Talked with Felix, said our README should be reflective of our plugin